src/controller/CHIPDeviceController.cpp

```diff
@@ -1631,6 +1649,57 @@ void AttributeReadFailure(void * context, CHIP_ERROR status)
     commissioner->CommissioningStageComplete(status);
 }
 
+void DeviceCommissioner::OnDone()
+{
+    NetworkClusters clusters;
+
+    CHIP_ERROR err = mAttributeCache->ForEachAttribute(
+        app::Clusters::NetworkCommissioning::Id, [this, &clusters](const app::ConcreteAttributePath & path) {
+            if (path.mAttributeId != app::Clusters::NetworkCommissioning::Attributes::FeatureMap::Id)
+            {
+                return CHIP_NO_ERROR;
+            }
+            TLV::TLVReader reader;
+            if (this->mAttributeCache->Get(path, reader) == CHIP_NO_ERROR)
```

Comment:
We can do a bit better than this.  We can do either:
```suggestion
            app::Clusters::NetworkCommissioning::Attributes::FeatureMap::TypeInfo::DecodableType flags;
            if (this->mAttributeCache->Get(path, flags) == CHIP_NO_ERROR)
            {
                BitFlags<app::Clusters::NetworkCommissioning::NetworkCommissioningFeature> features;
                // etc
```
or we can get the TLV reader and then do:
```
    BitFlags<app::Clusters::NetworkCommissioning::NetworkCommissioningFeature> features;
    if (app::DataModel::Decode(reader, features) == CHIP_NO_ERROR)
```
and directly decode the bitmap; that will do the SetRaw bits for us automatically.

Reply:
Good call - will update. I had it like that originally, but I thought it was failing to decode because I was getting back zeros. Turns out, I was really getting back zeros. Will fix this up in the next commit.

Reply:
k, changed to use decode into the features directly.